action.yml

```diff
@@ -99,23 +100,23 @@ runs:
         fi
         git checkout -b "$BRANCH_NAME"
 
-        echo "${{ steps.check-lts.outputs.latest-patch-version }}" > .go-version
+        echo "${{ steps.check-stable.outputs.latest-patch-version }}" > .go-version
 
         find . -name go.mod -execdir sh -c 'awk "/golang.org\/x/ {print \$1}" go.mod | grep "golang.org/x" | while read -r line; do go get $line@latest; done' \;
 
         find . -name go.mod -execdir go mod tidy \;
 
         git add .
-        git commit -m "Bump go version to ${{ steps.check-lts.outputs.latest-patch-version }}"
+        git commit -m "build(deps): bump go version to ${{ steps.check-stable.outputs.latest-patch-version }}"
         git push origin "$BRANCH_NAME"
 
     - name: 'Create PR to bump go version and stdlibs'
       shell: bash
-      if: steps.check-lts.outputs.lts == 'true' && steps.check-lts.outputs.needs-bump == 'true'
+      if: steps.check-stable.outputs.stable == 'true' && steps.check-stable.outputs.needs-bump == 'true'
       env:
         INPUT_REVIEWER: ${{ inputs.pr_reviewer }}
         GITHUB_TOKEN: ${{ inputs.github_token }}
       run: |
         BRANCH_NAME=${{ steps.git-push-changes.outputs.branch-name }}
-        gh pr create --base main --head "$BRANCH_NAME" --title "[auto] Bump go version to ${{ steps.check-lts.outputs.latest-patch-version }}" --body "Bump go version to ${{ steps.check-lts.outputs.latest-patch-version }}" --reviewer "${INPUT_REVIEWER}"
+        gh pr create --base main --head "$BRANCH_NAME" --title "[bump-go] Bump go version to ${{ steps.check-stable.outputs.latest-patch-version }}" --body 'Bump go version to [${{ steps.check-stable.outputs.latest-patch-version }}](https://go.dev/doc/devel/release#go${{ steps.get-go-version.outputs.go-major-version }}.minor)' --reviewer "${INPUT_REVIEWER}"
```

Comment:
~ Another small edit: use `[bump-go]` as the PR title prefix to indicate the tool, and include a doc link for the major version's minor releases